libnavigation-base/build.gradle

```diff
@@ -16,7 +16,6 @@ android {
 
 dependencies {
     api dependenciesList.mapboxCore
-    api dependenciesList.mapboxSdkServices
```

Comment:
Have we considered instead of removing `mapboxSdkServices` replacing it by `mapboxSdkGeoJSON` (`mapbox-sdk-geojson`)? That way we'll be able to keep using `Point` object (no need to replace every `Point` by `Location`, this https://github.com/mapbox/mapbox-navigation-android/pull/2261/files#diff-336bd43e2db2feb50e2c67ad2260f5e7R9 is not necessary either) + also as `mapboxSdkGeoJSON` includes Gson as a dependency https://github.com/mapbox/mapbox-java/blob/9bd6158292374fc6dd5f7eba33e7586ca1d76177/services-geojson/build.gradle#L9 `JsonMapper` https://github.com/mapbox/mapbox-java/blob/9bd6158292374fc6dd5f7eba33e7586ca1d76177/services-geojson/build.gradle#L9 changes introduced won't be necessary. How does that sound?

Reply:
@Guardiola31337 wdyt?

Reply:
I provided `Gson` default impl, please take a look

Reply:
@RingerJK let's stick to Gson. We're using it universally across SDKs. If we want to add an option to replace it, we should start with Mapbox Java and Maps SDK.

Reply:
ok, I see